packages/esm-ward-app/src/ward-workspace/ward-patient-notes/notes.resource.ts

```diff
@@ -0,0 +1,66 @@
+import { type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { type PatientNote, type UsePatientNotes, type VisitEncountersFetchResponse } from './types';
+import { type EncounterPayload } from '../../types';
+import { useMemo } from 'react';
+
+export function savePatientNote(payload: EncounterPayload, abortController: AbortController = new AbortController()) {
+  return openmrsFetch(`${restBaseUrl}/encounter`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
+    body: payload,
+    signal: abortController.signal,
+  });
+}
+
+export function usePatientNotes(patientUuid: string, encounterType: string, conceptUuid: string): UsePatientNotes {
+  const customRepresentation =
+    'custom:(uuid,display,encounterDatetime,patient,obs,' +
+    'encounterProviders:(uuid,display,' +
+    'encounterRole:(uuid,display),' +
+    'provider:(uuid,person:(uuid,display))),' +
+    'diagnoses';
+  const encountersApiUrl = `${restBaseUrl}/encounter?patient=${patientUuid}&encounterType=${encounterType}&v=${customRepresentation}`;
```

Comment:
Great... I will add the "visit" limiting factor once the endpoint is ready with it.